### ISSUE TITLE
Repetitive impact on palm or hand

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10670]
**Preferred term label:**
Repetitive impact on palm or hand

**Synonyms**
Use of vibratory tools (e.g., jackhammer)

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Carpal tunnel syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001106
- label: Exposure to repetive hand impact
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: A history of exposure to repetitive impact on palm or hand, such as working with a jackhammer.
- comment: This history can be a risk factor for carpal tunnel syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10670
- synonyms: Use of vibratory tools(NONE)
